code/handlers/trackservers.q

```diff
@@ -42,23 +42,19 @@ loadpassword:{
            [.lg.o[`conn;"password file ",(string file)," found"];
             .servers.USERPASS:first`$read0 hsym file]]};
     files:{.proc.getconfig["passwords/",(string x),".txt";2]} each `default,.proc.parentproctype,.proc.proctype,.proc.procname;
-    loadpassfile each (distinct files[;1],files[;0]) except `;
+    loadpassfile each (distinct files[;1],files[;0]) except`;
```

Comment:
Is `raze files[;1 0]` the same thing?

Reply:
They are the same but I will have to add something to sort the result so that `TorQ` files will be loaded before `TorQ-Finance` files

If you do `files[;1 0]`  you get:
```
/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ/config/passwords/default.txt /home/arisnoveanu/TorQProdSupp/TorQDev/TorQ-Finance-Starter-Pack/appconfig/passwords/default.txt
/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ/config/passwords/rdb.txt     /home/arisnoveanu/TorQProdSupp/TorQDev/TorQ-Finance-Starter-Pack/appconfig/passwords/rdb.txt
/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ/config/passwords/rdb1.txt    /home/arisnoveanu/TorQProdSupp/TorQDev/TorQ-Finance-Starter-Pack/appconfig/passwords/rdb1.txt
```

and if you raze it: 

```
`/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ/config/passwords/default.txt`/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ-Finance-Starter-Pack/appconfig/passwords/default.txt`/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ/config/passwords/rdb.txt`/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ-Finance-Starter-Pack/appconfig/passwords/rdb.txt`/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ/config/passwords/rdb1.txt`/home/arisnoveanu/TorQProdSupp/TorQDev/TorQ-Finance-Starter-Pack/appconfig/passwords/rdb1.txt
```

The order of the files is different and because these files have to be checked in order I will need to sort the list such that the `TorQ` files will be loaded before the `TorQ-Finance` files. 

I am not sure if it is an easy way to sort that list. I tried a few things but in my opinion it makes the code slightly more complicated and confusing then it is right now.

Reply:
what if you do `raze flip files[;1 0]` ?

Reply:
Try `flip` then `raze`?